### ISSUE TITLE
[BUG] - Random AP disconnect

### ISSUE THREAD

**Issue by henkhugo** [invernyx/md11-bugs#308]
### Describe the bug

Autopilot randomly disconnects

### Steps to reproduce

Cannot reproduce as it seems to be completely random. Possibly linked to MSFS turbulence realism?  

### Expected behaviour

to not disconnect 

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Crash

No

### Phase of flight

Cruise

### Departure

FALE

### Arrival

FACT

### Cruise level

FL340

### ZFW

415.35

### CG

21.7

### Fuel

62.45

### Cost index

102

### Route

FALE-FACT

### Location

Various 

### Agreement

- [X] Yes

**Comment by dr71205** [comment]
Might be input related, my AP was disconnecting because my joystick has a bit of jitter from a bad pot

**Comment by wagoozen1** [comment]
I expirience the same, after takeoff autopilots get disconnected after one second.
On all axis i applied a deadzone of 3%

**Comment by TheFightingRaven** [comment]
please refile if you can reproduce this issue

**Comment by Wise87** [comment]
I had two autopilot disconnects when aircraft had to make a slightly tight turns on the LNAV. 
I am flying the following. KPDX/10R LAVAA6 PDT DCT DLN DCT KU90Q DCT RAP DCT KD78A DCT KP69G DCT DSM DCT RAMSE FRIZN5 KSDF/35L.

When I hit the left turn at COSUG on the LAVAA6 the AP disconnected. When I made the right turn at LAVAA on the LAVAA6 I got a disconnect. Rest of flight was fine. No turbulence was noticed.
